src/adapters/node.js

```diff
@@ -8,7 +8,10 @@
         defer: Q.defer,
```

Comment:
The comment at the top of the file says that this file was generated from CoffeeScript, which is no longer true (or anyway we could not find the CoffeeScript source).  I suggest you remove the comment.

Reply:
Removed.